tck/README.md

```diff
@@ -116,6 +116,25 @@ In order to inform the TCK that your Publiher is not able to signal completion,
 }
 ```
 
+### Testing a "failed" Publisher
+The Reactive Streams spec mandates certain behaviours for Publishers which are "failed",
+e.g. it was unable to initialize a connection it needs to emit elements.
+It may be useful to specifically such known to be failed Publisher using the TCK.
+
+In order to run additional tests on your failed publisher you can implement the `createFailedPublisher` method.
+The expected behaviour from the returned implementation is to follow Rule 1.4 and Rule 1.9 - which are concerned
+with the order of emiting the Subscription and signaling the failure.
+
+```java
+@Override public Publisher<T> createFailedPublisher() {
+  final String invalidData = "this input string is known it to be failed";
+  return new MyPublisher(invalidData);
+}
+```
+
+In case you don't really have a known up-front error state you can put your Publisher into,
```

Comment:
Perhaps clarify that the Publish is not allowed to produce any onNexts, only onSubscribe + onError?